ft_freqanalysis.m

```diff
@@ -288,13 +288,13 @@
     
   case 'irasa'
     cfg.taper       = ft_getopt(cfg, 'taper', 'hanning');
-    cfg.output      = ft_getopt(cfg, 'output', 'pow');
+    cfg.output      = ft_getopt(cfg, 'output', 'fractal');
     cfg.pad         = ft_getopt(cfg, 'pad', 'nextpow2');
     if ~isequal(cfg.taper, 'hanning')
       ft_error('the irasa method supports hanning tapers only');
     end
-    if ~isequal(cfg.output, 'pow')
-      ft_error('the irasa method outputs power only');
+    if ~strcmp(cfg.output, {'fractal','original'})
```

Comment:
I suggest to move this check and error message to the `ft_spectest_irasa` function, which gets the desired `output` as  input argument.

Reply:
done!